fjord/__init__.py

```diff
@@ -1,7 +1,16 @@
+# First we import absolute imports from the future, so that our celery.py
+# module will not clash with the library
+from __future__ import absolute_import
+
 import os
 import site
 import sys
 
+from fjord.manage_utils import monkeypatch
```

Comment:
To be clear, this is probably not needed if I do `./manage.py celeryd`. But if I do `celery -A fjord worker -l info` it is. And that's the future!